### PR TITLE
test(components/indicators): add a11y testing coverage to tokens

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
[AB#2340624](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2340624)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1206/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1206/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1206/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1206/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1206/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1206/playground/)

File: libs/components/indicators/src/lib/modules/tokens/fixtures/token.component.fixture.html [hunk 1]
```diff
@@ -1 +1,8 @@
-<sky-token [ariaLabel]="ariaLabel"></sky-token>
+<sky-token
+  [ariaLabel]="ariaLabel"
+  [disabled]="disabled"
+  [dismissible]="dismissible"
+  [focusable]="foc
+"
```
Comment (johnhwhite) [inline libs/components/indicators/src/lib/modules/tokens/fixtures/token.component.fixture.html]: This seems like a typo.

Comment (Blackbaud-TrevorBurch) [reply libs/components/indicators/src/lib/modules/tokens/fixtures/token.component.fixture.html]: Huh - how did nothing catch that.... fixed.

**Review by johnhwhite — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See null

**Pull request merged**